box-share-sdk/src/main/java/com/box/androidsdk/share/fragments/DatePickerFragment.java

```diff
@@ -49,6 +49,10 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
         return mDialog;
     }
 
+    public void setOnDateSetListener(DatePickerDialog.OnDateSetListener listener) {
```

Comment:
Remember to setOnRetainInstanceState on this otherwise a rotation will probably dismiss this fragment.